api/src/main/java/jakarta/data/repository/Insert.java

```diff
@@ -25,54 +25,45 @@
 import java.lang.annotation.Target;
 
 /**
- * <p>Annotates a repository method that inserts entities.</p>
+ * <p>Lifecycle annotation for repository methods which perform insert operations.</p>
  *
- * <p>The {@code Insert} annotation indicates that the annotated repository method requests that one or more entities
- * be inserted into the database. This method must have a single parameter whose type must be one of the following:
+ * <p>The {@code Insert} annotation indicates that the annotated repository method adds the state of one or more
+ * entities to the database. The annotated repository method usually has exactly one parameter whose type is one of
```

Comment:
The more permissive language is not need for Insert.
```suggestion
 * entities to the database. The annotated repository method must have exactly one parameter whose type is one of
```

Reply:
True, good point, I will fix that.